app/components/footer_component.html.erb

```diff
@@ -1,35 +1,98 @@
+        <p class="small">Your details are protected under the terms of our <%= link_to "privacy notice", privacy_policy_path, class: "link" %> . Our privacy notice explains how we use your personal data. It's important you read this before chatting to us.</p>
+      </div>
+    </section>
+
+    <section>
+      <hr class="site-footer__divider">
+
+      <div class="site-footer__links-container">
+        <div class="site-footer__links-column">
+          <h3 class="heading-m">Become a teacher</h3>
+          <ul>
+            <li><a class="link--black" href="/steps-to-become-a-teacher">How to become a teacher</a></li>
+            <li><a class="link--black" href="/train-to-be-a-teacher/qualifications-you-need-to-teach">Qualifications needed to teach</a></li>
+            <li><a class="link--black" href="/train-to-be-a-teacher">Teacher training</a></li>
+            <li><a class="link--black" href="/train-to-be-a-teacher/what-is-qts">Qualified teacher status (QTS)</a></li>
+            <li><a class="link--black" href="/life-as-a-teacher/pay-and-benefits/teacher-pay">Teacher salary</a></li>
+          </ul>
+        </div>
+        <div class="site-footer__links-column">
+          <h3 class="heading-m">Finance and funding</h3>
+          <ul>
+            <li><a class="link--black" href="/funding-and-support/scholarships-and-bursaries">Bursaries and scholarships</a></li>
+            <li><a class="link--black" href="/funding-and-support/salaried-teacher-training">Salaried teacher training</a></li>
+            <li><a class="link--black" href="/funding-and-support/tuition-fees">Tuition fees</a></li>
+            <li><a class="link--black" href="/funding-and-support/tuition-fee-and-maintenance-loans">Student finance</a></li>
+            <li><a class="link--black" href="/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme">Teacher pensions</a></li>
+          </ul>
+        </div>
+        <div class="site-footer__links-column">
+          <h3 class="heading-m">Routes into teaching</h3>
+          <ul>
+            <li><a class="link--black" href="/train-to-be-a-teacher/if-you-have-a-degree">With a degree</a></li>
+            <li><a class="link--black" href="/train-to-be-a-teacher/if-you-dont-have-a-degree">Without a degree</a></li>
+            <li><a class="link--black" href="/train-to-be-a-teacher/assessment-only-route-to-qts">Assessment only QTS</a></li>
+            <li><a class="link--black" href="/life-as-a-teacher/teaching-as-a-career/change-careers">Retrain as a teacher</a></li>
+            <li><a class="link--black" href="/train-to-be-a-teacher/teacher-degree-apprenticeships">Teacher degree apprenticeship</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <hr class="site-footer__divider">
+    </section>
+
+    <div class="site-footer__social-icons">
+      <a href="https://www.facebook.com/getintoteaching" class="site-footer-top__social__link" target="_blank" rel="noopener">
```

Comment:
Social media icons disappear in active state (black on black) @jenhadfield-dfe 

![image](https://github.com/user-attachments/assets/b62f11a2-7707-4569-b898-51d777bbdddf)

Reply:
@spencerldixon Would you be able to add the reverse social media icons for the active state, so they are visible when tabbed to? We have the white versions already in production on the black footer

Reply:
Done!

Reply:
@spencerldixon This isn't quite working still - icons are still not visible when tabbed to using keyboard only:

![image](https://github.com/user-attachments/assets/ff267c97-da0f-4576-92bc-290c40877533)

Reply:
Icons look good now - resolving this - thanks!